src/components/PurchaseOrder/POForm.js

```diff
@@ -193,6 +193,7 @@ class POForm extends Component {
     const vendors = get(parentResources, 'vendors.records', [])
       .filter(vendor => vendor.isVendor && vendor.status === ORGANIZATION_STATUS_ACTIVE);
     const orderTemplates = getOrderTemplatesForSelect(parentResources);
+    const poLinesLength = get(initialValues, 'compositePoLines', '').length;
```

Comment:
but it's an array, why do you use empty string as a default value?

Reply:
Thanks, it was misprint, i'll fix it